### PR TITLE
Added Silences SA to automate silencing clusters

### PR DESCRIPTION
**Pull request by T-Kukawka**
Goal of this PR is to create the SA, ClusterRole and ClusterRoleBinding in order for customers to be able to automate creation of silences and needed PSP following the: https://docs.giantswarm.io/advanced/cost-optimization/workload-clusters-scale-down/

Team issue: https://github.com/giantswarm/giantswarm/issues/22108

### CODE REVIEW

**Comment by marians** [comment]
A linked issue and some description in the PR would help a lot. 💐

**Comment by T-Kukawka** [comment]
@marians added the description/issue

**Comment by marians** [comment]
Looks good to me if it works.

So the general idea is that we are creating specific "automation" service accounts which are already prepared with the right permissions from the beginning?

**Review by marians — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+
+- Added Silences Automation account
```
Comment (marians) [inline CHANGELOG.md:12]: Maybe use the actual SA name here?

**Comment by T-Kukawka** [comment]
@marians 

> Looks good to me if it works.
> 
> So the general idea is that we are creating specific "automation" service accounts which are already prepared with the right permissions from the beginning?

Yes - there is still need to create a PSP but i can move it to the terraform repo as a next step - wanted to ship this first - after i test and realease will update docs and iterate on this

**Comment by T-Kukawka** [comment]
Changing the setup/approach here 😓

**Pull request closed without merging**